bin/common.c

```diff
@@ -314,6 +314,8 @@ int s2n_setup_external_psk_list(struct s2n_connection *conn, char *psk_optarg_li
 
 int s2n_set_common_server_config(int max_early_data, struct s2n_config *config, struct conn_settings conn_settings, const char *cipher_prefs, const char *session_ticket_key_file_path)
 {
+    GUARD_EXIT(s2n_config_set_max_blinding_delay(config, 0), "Error setting blinding delay");
```

Comment:
Currently s2nc and s2nd are our only standalone client/server examples in C. Is there any concern about users unknowingly copy pasting this in their applications? I wonder if a comment could potentially help to avoid this?
```suggestion
    /* The s2n-tls blinding security feature is disabled for testing purposes to make debugging easier. */
    GUARD_EXIT(s2n_config_set_max_blinding_delay(config, 0), "Error setting blinding delay");
```